src/rust/rust-proto/src/graplinc/grapl/api/plugin_registry/v1beta1_client.rs

```diff
@@ -41,6 +43,15 @@ pub enum PluginRegistryServiceClientError {
     Elapsed,
 }
 
+// This likely should be removed in favor of making all SerDe conversions
+// TryFrom<Error = SerDeError> instead of From.
+// Currently it's kind of scattershot.
```

Comment:
It's not scattershot. The things that are fallible are `TryFrom` and the things that are infallible are `From`. If this code needs to paper over that, then this is a fine approach. I don't think we should compromise the clarity of the `From`/`TryFrom` distinction in the actual SerDe types to accommodate things like this though.